src/go.mod

```diff
@@ -59,6 +59,7 @@ require (
 	github.com/opentracing/opentracing-go v1.1.0 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/pquerna/cachecontrol v0.0.0-20180517163645-1555304b9b35 // indirect
+	github.com/prometheus/client_golang v1.0.0
```

Comment:
Is there a reason to pin to v1.0.0? I see the recent release is v1.8.0:
https://github.com/prometheus/client_golang/releases